test/chemistry/potential_energy_models/lennard_jones_potential_test.py

```diff
@@ -26,7 +26,7 @@ def test_args_fixture():
 
 def test_basic_law(test_args):
     result = lennard_jones_potential.calculate_potential(test_args.e, test_args.sigma, test_args.r)
-    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.energy)
+    assert equivalent_dims(result, units.energy)
```

Comment:
Revert this change. We will introduce it later.

Reply:
Oh I see your new functionality, okay